widgets/features/styles/default.less

```diff
@@ -36,6 +34,14 @@
         padding: 0 25px;
         display: flex;
 
+        &:nth-of-type(@{per_row}n+1) {
+            padding-left: 0;
+        }
+
+        &:last-of-type, &:nth-of-type(@{per_row}n) {
```

Comment:
This `last-of-type` selector will result in the icon being misaligned inside the `.sow-features-feature` container if you don't have an exact multiple of the number of features per row and it didn't seem to be necessary in my quick tests.